### PR TITLE
feat(Docs): Docs release notes June 21, 2024

### PR DESCRIPTION
**Pull request by paperclypse**

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+### New docs
+
+* We consolidated all of the errors inbox mobile crash docs into a single [mobile group crashes tab](/docs/errors-inbox/mobile-tab).
+* New [Analyze model data](/docs/ai-monitoring/explore-ai-data/view-model-data) describes the different ways you can use AI monitoring to observe your model performance. 
+* New [Google VertexAI monitoring integration](/docs/infrastructure/google-cloud-platform-integrations/gcp-integrations-list/google-vertexai-monitoring-integration) describes how to activate the integration and start monitoring it.
+* New [NVIDIA Jetson integration](/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration) describes how to monitor your NVIDIA Jetson device.
+
+### Major changes
+
+* In our Python agent configuration, added [startup debug](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#NEW_RELIC_STARTUP_DEBUG) info.
```
Comment (akristen) [inline src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:16]:
```suggestion
* In our Python agent configuration, added [`STARTUP_DEBUG`](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#NEW_RELIC_STARTUP_DEBUG) info.
```

File: src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* In [mobile monitoring configure settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-background-reporting), added enable/disable info for background and `ApplicationExitInfo` reporting for Android.
```
Comment (akristen) [inline src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:17]:
```suggestion
* In [mobile monitoring configure settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-background-reporting), added information to enable or disable background reporting, and added `ApplicationExitInfo` reporting for Android.
```

Comment (akristen) [reply src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:17]: i think we want to move away from using `/` in our docs, last i heard in doc_talk. not confident on that, but added an alternative wording to sidestep that

File: src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* A massive overhaul of the alerts navigation makes it easier for you to find what you're looking for in those docs.
+
+### Minor changes
+
+* Updated [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps) and [Android](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps) app launch time docs in response to customer feedback to make them more similar to one another.
```
Comment (akristen) [inline src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:22]:
```suggestion
* Aligned the structure of [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps) and [Android](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps) app launch time docs in response to customer feedback, which suggested we make them more similar.
```

Comment (akristen) [reply src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:22]: i think we can cut the ", which" clause tbh

File: src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+* [Infrastructure logs](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent) now support Ubuntu 24.04.
```
Comment (akristen) [inline src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:23]:
```suggestion
* [Infrastructure logs](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent) now support Ubuntu 24.04 (Noble Numbat).
```

Comment (akristen) [reply src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:23]: my weird niche knowledge about ubuntu and learning how to troubleshoot ubuntu on my VM taught me that they refer to all their updates by """codenames""" that users like to use

File: src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:17]: fair enough!

File: src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx (hunk 4 above)
Comment (akristen) [reply src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:23]: we also refer to them in our ubuntu docs too!

Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:23]: Good call!

File: src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+* New [Analyze model data](/docs/ai-monitoring/explore-ai-data/view-model-data) describes the different ways you can use AI monitoring to observe your model performance. 
+* New [Google VertexAI monitoring integration](/docs/infrastructure/google-cloud-platform-integrations/gcp-integrations-list/google-vertexai-monitoring-integration) describes how to activate the integration and start monitoring it.
+* New [NVIDIA Jetson integration](/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration) describes how to monitor your NVIDIA Jetson device.
+
+### Major changes
+
+* In our Python agent configuration, added [startup debug](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#NEW_RELIC_STARTUP_DEBUG) info.
+* In [mobile monitoring configure settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-background-reporting), added enable/disable info for background and `ApplicationExitInfo` reporting for Android.
+* A massive overhaul of the alerts navigation makes it easier for you to find what you're looking for in those docs.
```
Comment (akristen) [inline src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx:18]:
```suggestion
* A massive overhaul of the alerts navigation updated the docs to map cleanly onto the alerts user journey, guiding users through alerts creation, configuration, and management```

**Review by akristen — APPROVED** [review]
LGTM

**Pull request merged**